File: Source/Fantasy.Jobs/Scheduling/Action.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Linq;
using System.Runtime.Serialization;
using Fantasy.XSerialization;

namespace Fantasy.Jobs.Scheduling
{
    [DataContract]
    [XSerializable("action", NamespaceUri=Consts.ScheduleNamespaceURI)]
    public abstract class Action
    {
        public abstract  ActionType Type { get; }
    }
}

File: AutoConverted/Fantasy.Jobs/Scheduling/Action.java
package Fantasy.Jobs.Scheduling;

import Fantasy.XSerialization.*;

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
//[DataContract, XSerializable("action", NamespaceUri=Consts.ScheduleNamespaceURI)]
public abstract class Action
{
	public abstract ActionType getType();
}